### ISSUE TITLE
Account toggle does not work on Safari 10.1

### ISSUE THREAD

**Issue by HvandenBerg** [wagtail/wagtail#4646]
### Issue Summary

Clicking the account toggle does not open the account settings / logout.
![actual](https://user-images.githubusercontent.com/25935102/41656949-f3a1b88a-7491-11e8-87f0-a27bf3d5fcd7.gif)
![expected](https://user-images.githubusercontent.com/25935102/41656951-f3cd39b0-7491-11e8-97e8-5cf1d89eb50f.gif)

###Version: 1.11.5roduce
Open Wagtail
login
click on username / icon to toggle to open toggle to  'account settings / Logout'
This does not open / icon does change

### Technical details

* Python version: ~~2.7.10~~ 3.6.3
* Django version: Version: 1.11.5
* Wagtail version: Wagtail v.2.2a0
* Browser version: ~~Pretty much every browser I've tried that is supported~~ Safari 10.1 on Browserstack

**Comment by gasman** [comment]
Hi @HvandenBerg - thanks for the report. This is working fine on my local development instance running against Django 1.11.5 and Python 3.4.3 (Wagtail 2 is not compatible with Python 2.7.x), so I suspect this is an installation issue.

A couple of questions:

* Have you cleared your browser cache?
* Does this happen on a fresh installation of Wagtail?
* How did you install Wagtail? (If you installed from git, did you run the `npm run build` step to build the js/css assets?)
* Are there any Javascript errors in the browser console?

**Comment by HvandenBerg** [comment]
Hi @gasman. After checking with @jjanssen I've run 'npm run build' on the wagtail master within the bakery demo and this solved the issue for most browsers. However it does still occur on Safari 10.1 on Browserstack. I've made sure to clear all cache before retesting and there are no errors in the browser console. When I log in and click the account toggle nothing happens, after 3 clicks I do see the desired effect (account menu toggle).

Also, apparently I checked the python version in the wrong place and it is indeeld version 3.6.3. I'll update the issue to reflect this.

Finally I checked the behaviour in another project running wagtail version 1.13.1 and this works fine.
